<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettings</class>
 <widget class="QWidget" name="AppearanceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="checkHomeIcon_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblMountedIcon_4">
       <property name="text">
        <string>Screen Keyboard</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="checkTrashIcon">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="lblMountedIcon_3">
       <property name="text">
        <string>Login Mode</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="checkShowDesktopIcons">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Show Desktop Icons</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="checkHomeIcon">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lblTrashIcon">
       <property name="text">
        <string>Trash Icon</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblMountedIcon">
       <property name="text">
        <string>Mounted Volumes Icon</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblMountedIcon_2">
       <property name="text">
        <string>Desktop Mode</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QComboBox" name="comboBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblHomeIcon">
       <property name="text">
        <string>Home Icon</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="checkMountedVulmesIcon">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label_4">
       <property name="accessibleName">
        <string>title</string>
       </property>
       <property name="text">
        <string>Icons</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="QLabel" name="label_2">
       <property name="accessibleName">
        <string>title</string>
       </property>
       <property name="text">
        <string>Background Image Mode</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QLabel" name="label_3">
       <property name="accessibleName">
        <string>title</string>
       </property>
       <property name="text">
        <string>Screen Applications</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="lblMountedIcon_5">
       <property name="text">
        <string>Screen Reader</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="checkHomeIcon_3">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="lblNetworkIcon">
       <property name="text">
        <string>Network Icon</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="checkNetworkIcon">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
